nslsii/common/ipynb/logutils.py

```diff
@@ -34,6 +35,9 @@ def log_exception(ipyshell, etype, evalue, tb, tb_offset=None):
     tb_lines = traceback.format_exception(etype, evalue, tb)
     for tb_line in tb_lines:
         ipyshell.logger.log_write(tb_line, kind="output")
-    ipyshell.showtraceback((etype, evalue, tb), tb_offset=tb_offset)
+    ipyshell.showtraceback((etype, evalue, tb), tb_offset=tb_offset, exception_only=True)
+
+    logging.getLogger("bluesky.ipython").exception(evalue)
+    print("see bluesky log file for full stack trace")
```

Comment:
I believe that logging goes to the stderr, and this should go with it. (The mostly likely user-facing practical consequence of that is that Jupyter notebooks format stderr output differently.)

```suggestion
    print("see bluesky log file for full stack trace", file=sys.stderr)
```

Reply:
I agree.